packages/components/organisms/f-loyalty/CHANGELOG.md

```diff
@@ -13,6 +13,14 @@ v0.4.0
 - Unit tests for how it works page
 
 
+Latest (to be added to next release)
```

Comment:
Nit: this too

Reply:
Same as above – will do as part of follow-up PR